common/test/acceptance/tests/lms/test_lms_edxnotes.py

```diff
@@ -11,7 +10,6 @@
 from ...fixtures.edxnotes import EdxNotesFixture, Note, Range
 
 
-@skipUnless(os.environ.get("FEATURE_EDXNOTES"), "Requires Student Notes feature to be enabled")
```

Comment:
I hadn't noticed that this was not the setting but was a separate environment variable. It seems very odd that there are two different ways to enable the feature. Can we remove one of them?